src/components/DataTypes/index.ts

```diff
@@ -8,6 +8,14 @@ import * as objectType from './object';
 import * as oneOfType from './oneOf';
 import * as stringType from './string';
 
+export interface DataTypeEditProps {
+	schema: JSONSchema;
+	value?: any;
+	onUpdate: (value: string | number | boolean) => void;
+	operator: string;
+	slim?: boolean;
+}
+
```

Comment:
Is this still necessary? I don't see it being used anywhere

Reply:
🙏